provider/provider_v6.go

```diff
@@ -19,7 +19,7 @@ import (
 
 const (
 	clientConcurrency = 15
-	defaultTimeout    = 60
+	defaultTimeout    = 220
```

Comment:
why is that needed?

Reply:
Sorry, forgot to remove the local change